Magical Nights/items/enchanted.json

```
@@ -4,20 +4,60 @@
     "type": "GENERIC",
     "symbol": "/",
     "color": "brown",
+    "name": { "str": "minor staff of the magi", "str_pl": "minor staves of the magi" },
+    "description": "A simple carved staff, made of wood.  It faintly glows with magic when you cast spells.  Wielding it slightly increases your mana regeneration",
```

```suggestion
    "description": "A simple carved staff, made of wood.  It faintly glows with magic when you cast spells.  Wielding it slightly increases your mana capacity",
```
Pretty sure mana_cap is the capacity, rather than regeneration